.github/workflows/publish-rna.yml

```diff
@@ -0,0 +1,17 @@
+# Description: This workflow runs unit + e2e tests
+#
+# Triggered by: merge to `rna/release` branch
+
+name: Publish / RNA
+
+on:
+  push:
+    branches: [rna/release]
+
+jobs:
+  publish:
+    uses: ./.github/workflows/reusable-tagged-publish.yml
+    with:
+      dist-tag: rn-authenticator
```

Comment:
Should we just name this `rna`?

Reply:
Thought abbreviaiton `rna` was somewhat confusing for audiences without context, but open to thoughts.

Reply:
Since this for internal testing, think `rna` is totally fine

Reply:
```suggestion
      dist-tag: rna
```